packages/block-library/src/cover/edit/index.js

```diff
@@ -46,6 +47,18 @@ import ResizableCover from './resizable-cover';
 
 extend( [ namesPlugin ] );
 
+const placeholderIllustration = (
```

Comment:
Do we also have this in the featured image block? We should share the same SVG code probably.

Reply:
Let's copy it for now. I wouldn't even know where to export it from.